src/core/src/main/java/io/github/lonamiwebs/stringlate/classes/resources/tags/ResTag.java

```diff
@@ -88,8 +88,8 @@ public int compareTo(final ResTag o) {
     //region De/sanitize Content
 
     private static boolean isEscapeSequence(char which) {
-        for (int i = 0; i < ESCAPE_SEQUENCES.length; ++i) {
-            if (ESCAPE_SEQUENCES[i] == which) {
+        for (char ESCAPE_SEQUENCE : ESCAPE_SEQUENCES) {
```

Comment:
While we're at it, `ESCAPE_SEQUENCE` should be `escapeSequence`, really. This isn't a constant.